splice_access_api/src/main/java/com/splicemachine/access/configuration/SIConfigurations.java

```diff
@@ -155,7 +155,7 @@
 
     // Threads processing first queue
     public static final String ROLLFORWARD_FIRST_THREADS = "splice.txn.rollforward.firstQueueThreads";
-    public static final int DEFAULT_ROLLFORWARD_FIRST_THREADS = 10;
+    public static final int DEFAULT_ROLLFORWARD_FIRST_THREADS = 25;
```

Comment:
Do we need that many roll forward threads? Have you tried decreasing the queue wait times, since we are now getting better latencies?

Reply:
After the change, roll forward threads are 100% busy when tested with 10, 20, 40 threads. 
TPC-C benchmark performance is sensitive to their throughput (mostly for the second and third hour). The 25 value is a compromise to get better current performance before we understand the new behavior better.